pkg/shed/vector_uint64.go

```diff
@@ -66,16 +66,19 @@ func (f Uint64Vector) Put(i, val uint64) (err error) {
 
 // PutInBatch stores a uint64 value at index i in a batch
 // that can be saved later in the database.
-func (f Uint64Vector) PutInBatch(batch *leveldb.Batch, i, val uint64) {
-	batch.Put(f.indexKey(i), encodeUint64(val))
+func (f Uint64Vector) PutInBatch(batch *badger.Txn, i, val uint64) {
+	err := batch.Set(f.indexKey(i), encodeUint64(val))
+	if err != nil {
+		f.logger.Debugf("could not put uint64 value in batch. Error : ", err.Error())
```

Comment:
Again, error is not returned, which may result in data inconsistencies if the batch is committed, as the error is not propagated.

Reply:
Do not log error and return it.